contracts/ProtocolGovernance.sol

```diff
@@ -130,125 +56,60 @@ contract ProtocolGovernance is IProtocolGovernance, DefaultAccessControl {
         return params.protocolTreasury;
     }
 
-    // -------------------  EXTERNAL, MUTATING, GOVERNANCE, DELAY  -------------------
+    // ------------------- PUBLIC, MUTATING, GOVERNANCE, IMMEDIATE -----------------
 
-    /// @inheritdoc IProtocolGovernance
-    function setPendingClaimAllowlistAdd(address[] calldata addresses) external {
-        require(isAdmin(msg.sender), ExceptionsLibrary.FORBIDDEN);
-        _pendingClaimAllowlistAdd = addresses;
-        pendingClaimAllowlistAddTimestamp = block.timestamp + params.governanceDelay;
+    function rollbackStagedPermissions() external {
```

Comment:
I'd move it to AddressPermission control (which inherits DefaultAccessControl) and thus make this contract self-contained